docs/sources/operator/deploy-agent-operator-resources.md

```diff
@@ -58,7 +58,6 @@ To deploy the `GrafanaAgent` resource:
         selector:
           matchLabels:
               agent: grafana-agent-integrations
-      image: grafana/agent:v0.34.1
```

Comment:
I think I made a conflict with the latest main commit. Can you merge main in again? Thanks.